lib/rubocop/cop/mixin/regexp_literal_help.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    # Common functionality for handling Regexp literals.
+    module RegexpLiteralHelp
```

Comment:
I'll merge this as is, because I'm planning to cut a release today, but this has to be moved to a note extension in `rubocop-ast`.